helm/ako/values.yaml

```diff
@@ -93,6 +93,7 @@ L7Settings:
   passthroughShardSize: "SMALL" # Control the passthrough virtualservice numbers using this ENUM. ENUMs: LARGE, MEDIUM, SMALL
   enableMCI: "false" # Enabling this flag would tell AKO to start processing multi-cluster ingress objects.
   fqdnReusePolicy: "InterNamespaceAllowed" # Use this to control whether AKO allows cross-namespace usage of FQDNs. enum Strict|InterNamespaceAllowed
+  defaultRouteDomain: "" # If spec.subdomain field is specified instead of spec.host field for an OpenShift route, then defaultRouteDomain is appended to the subdomain to form the FQDN.
```

Comment:
It will be good if we provide an example of subdomain to clarify the format. Example, AKO expect `avi.internal` as subdomain and not `.avi.internal`.

Reply:
Is it possible to use `L4Settings.defaultDomain` field? It serves the same purpose. We can move this field to `Networksetting` or another section of values.yaml so that it will represent both context L4 and `L7 in openshift`?